cmd/wallet-web/test/fixtures/testdata/contexts/authorization-credential-v1.json

```diff
@@ -0,0 +1,40 @@
+{
```

Comment:
why we need these contexts? shouldn't be added to this [file](https://github.com/trustbloc/wallet/blob/main/test/fixtures/wallet-web/static/contexts/ld-contexts.json) instead?

Reply:
I'll add them to [this](https://github.com/trustbloc/wallet/blob/main/cmd/wallet-web/test/fixtures/testdata/contexts/agent-startup-contexts.json) instead, since the tests using them are in wallet-web unit tests.